packages/react/src/components/TableOfContents/TableOfContents.js

```diff
@@ -26,10 +27,11 @@ const { prefix } = settings;
  *
  * @param {object} props props object
  * @param {object} props.menuItems menu items object
+ * @param {string} props.theme selected theme string
  * @param {*} props.children children property of component
  * @returns {*} JSX Object
  */
-const TableOfContents = ({ menuItems, children, menuLabel }) => {
+const TableOfContents = ({ menuItems, children, theme }) => {
```

Comment:
are we not using `menuLabel` anymore?

Reply:
We need it for mobile

<img width="486" alt="Screen Shot 2019-12-06 at 2 31 19 PM" src="https://user-images.githubusercontent.com/54281166/70350659-23223880-1835-11ea-91a5-ecc284bb1e53.png">

Reply:
We need it for mobile

<img width="486" alt="Screen Shot 2019-12-06 at 2 31 19 PM" src="https://user-images.githubusercontent.com/54281166/70350659-23223880-1835-11ea-91a5-ecc284bb1e53.png">